### DISCUSSION TITLE
Is it possible to get network bytes received/sent per process ?

### DISCUSSION THREAD

**Discussion by facewindu** [oshi/oshi#2715] [General]
We have for instance access to `OsProcess#getBytesRead` / `OsProcess#getBytesWritten` to get the `HWDiskStore` data per process.
Is it possible to get the info otherwise available via `NetworkIF#getBytesRecv` / `NetworkIF#getBytesSent` per process, i.e. to know how a given process accessed network interfaces ?

**Comment by dbwiddis** [comment]
I don't think this data is even collected at the operating system level without software actively monitoring it.  Pretty sure there's an issue somewhere with a similar question but didn't find it in a short search.
